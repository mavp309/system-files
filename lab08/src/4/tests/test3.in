-4.56 -1.11 3.3 5.0
