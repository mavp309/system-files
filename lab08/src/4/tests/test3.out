-2.0600 0.5400
